plugin/src/main/kotlin/com/github/triplet/gradle/play/ProcessPackageMetadataTask.kt

```diff
@@ -0,0 +1,30 @@
+package com.github.triplet.gradle.play
+
+import com.android.build.gradle.api.ApkVariantOutput
+import com.github.triplet.gradle.play.internal.PlayPublishTaskBase
+import com.github.triplet.gradle.play.internal.ResolutionStrategy
+import org.gradle.api.tasks.TaskAction
+
+open class ProcessPackageMetadataTask : PlayPublishTaskBase() {
+    @TaskAction
+    fun processVersionCodes() = read { editId ->
+        val maxVersionCode = bundles().list(variant.applicationId, editId).execute().bundles
+                ?.map { it.versionCode }
+                ?.max() ?: apks().list(variant.applicationId, editId).execute().apks
+                ?.map { it.versionCode }
+                ?.max() ?: 1
```

Comment:
Shouldn't the unfound `maxVersionCode` be either `0` or `null`? In practice, it isn't going to be unfound for anything that we can upload, though.

Reply:
I'm using 1 because if the API lets us upload the first APK without using the console, 1 is the default. But yeah, mostly it's just to make the Kotlin type system happy.